pkg/cloudprovider/cloud.go

```diff
@@ -62,7 +62,7 @@ type TCPLoadBalancer interface {
 	// GetTCPLoadBalancer returns whether the specified load balancer exists, and
 	// if so, what its IP address or hostname is.
 	GetTCPLoadBalancer(name, region string) (endpoint string, exists bool, err error)
-	// CreateTCPLoadBalancer creates a new tcp load balancer. Returns the IP address or hostname of the balancer
+	// CreateTCPLoadBalancer idempotently creates or updates a tcp load balancer. Returns the IP address or hostname of the balancer
```

Comment:
Having Create handle updating as well seems like a high bar - one that certainly won't work with the current GCE implementation. Is there a reason that it's needed vs Create only being responsible for the initial creation and relying on the service controller to handle the necessary update logic?